lib/signs_ui_web/controllers/auth_controller.ex

```diff
@@ -52,6 +52,35 @@ defmodule SignsUiWeb.AuthController do
     end
   end
 
+  @spec logout(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def logout(conn, _params) do
+    refresh_token_store = Application.get_env(:signs_ui, :refresh_token_store)
+
+    conn
+    |> Plug.Conn.fetch_session()
+    |> Plug.Conn.get_session(:signs_ui_username)
+    |> refresh_token_store.clear_refresh_token()
+
+    auth_domain =
+      :ueberauth |> Application.get_env(Ueberauth.Strategy.Cognito) |> Keyword.get(:auth_domain)
+
+    redirect_params =
+      URI.encode_query(%{
+        "client_id" =>
+          :ueberauth
+          |> Application.get_env(Ueberauth.Strategy.Cognito)
```

Comment:
Should this be using some information about the ueberauth provider, rather than hardcoding Cognito?

Reply:
I just pushed up a commit that should make it a bit more extensible if we were to add / change providers.